scripts/ts-to-ts49.ts

```diff
@@ -1,96 +0,0 @@
-/* eslint-disable @typescript-eslint/no-shadow,no-restricted-syntax */
```

Comment:
I assume you're deleting this file because it's no longer used/ran?

Please add this to the PR description.